### ISSUE TITLE
Rewrite the computer vision examples using webgl

### ISSUE THREAD

**Issue by stuhlmueller** [probmods/webppl#79]
Computer vision examples: http://dippl.org/examples/vision.html

Rewrite either directly in webgl (see, e.g., [webglfundamentals.org](http://webglfundamentals.org/)) or by using a library like [three.js](http://threejs.org/) or [pixi.js](http://www.pixijs.com/). Then compare performance to the plain canvas implementation. What happens if we increase image size? 

We could also try implementing a simple 3D analog to the 2D rectangles example.

**Comment by ngoodman** [comment]
Yes, this would be worth doing. I think that @dritchie probably has (3d) webgl working in the procedural-modeling branch.

**Comment by dritchie** [comment]
That branch just has the modifications I made to MH and ParticleFilter. The actual meat of the code is in a different repository: https://github.com/dritchie/web-procmod.
